src/lib.rs

```diff
@@ -1458,4 +1465,22 @@ mod test {
         assert_eq!(a[&2], "two");
         assert_eq!(a[&3], "three");
     }
+
+    // test serde trait implementations
```

Comment:
An even simpler approach would be to replace all the code from here onward with:

```rust
#[test]
#[cfg(feature = "eders")]
fn test_serde() {
    use serde::{Serialize, Deserialize};

    fn impls_serde_traits<S: Serialize + Deserialize>() {}

    impls_serde_traits::<VecMap<u32>>();
}
```